akka-actor/src/main/resources/reference.conf

```diff
@@ -25,6 +25,17 @@ akka {
   # (akka.actor.ActorSystem.Settings, akka.event.EventStream) parameters.
   logging-filter = "akka.event.DefaultLoggingFilter"
 
+  # Specifies the default loggers dispatcher
+  loggers-dispatcher = "akka.loggers.pinned-dispatcher"
```

Comment:
I meant `akka.io.pinned-dispatcher` happy to force push if that's what you want, it would only point to that other dispatcher.